amethyst_assets/src/loader.rs

```diff
@@ -163,10 +163,11 @@ impl Loader {
     fn context<C>(&self) -> &Arc<C>
         where C: Context + 'static,
     {
-        let context = self.contexts
+        let context: &Box<Any + Send + Sync> = self.contexts
             .get(&TypeId::of::<C::Asset>())
             .expect("Assets need to be registered with `Loader::register`.");
 
+        let context: &Any = context; // `Any + Send + Sync` doesn't has `downcast_ref`
```

Comment:
[Any](https://doc.rust-lang.org/beta/std/any/trait.Any.html) is a trait per the documentation.